tinkerbell/stack/values.yaml

```diff
@@ -29,46 +29,27 @@ stack:
     imagePullPolicy: IfNotPresent
     roleName: kube-vip-role
     roleBindingName: kube-vip-rolebinding
+    interface: eth0
```

Comment:
do we want to include this by default? without it kubevip will auto detect the interface. maybe comment it out with a comment about its use?

Reply:
Yes, thats a good idea. Thanks.

Reply:
Done.